OasisDEX/oasis-borrow
# 🧿 Ajna Earn pool selector

# 🧿 Ajna Earn pool selector

For now selector is using dummy `ajnaPoolDummyData` data with dummy `AjnaPoolData` type, but I think it's safe to assume that in the end data we are going to get from some observable form library is going to be of similar type, so no time wasted.
  
## Changes 👷‍♀️

- Added support for Earn in Ajna pool selector.
  
## How to test 🧪

- See if `ajna/earn` looks correct,
- do not click `Earn` button, I was to afraid to see what is going to happen.
